### ISSUE TITLE
 After invoking the 'Columns' control, Keyboard tab focus order is not in sequential while navigating using TAB key.

### ISSUE THREAD

**Issue by msftedad** [ag-grid/ag-grid#6272]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```
**Providing a Reproducible Scenario**
Repro Steps
1.Open URL:https://codesandbox.io/s/react-functional-component-forked-o0tgbf 
2.Keyboard focus automatically moves on ‘index.js’ field.
4.Now Press ‘Ctrl+M' key then Navigate to 'Column' control by using Tab key.
5.Invoke 'column' control by pressing Enter and navigate through the pop-up by using Tab key and observe the focus order is sequential or not.

**Current behavior**
After invoking the 'Columns' control, Keyboard tab focus order is not in sequential while navigating using TAB key.

Observation: After invoking 'Columns' control, Keyboard tab focus is moving as below
Invoke columns control->columns popup displays->Filters control->Check box and edit field in popup->chevron icon->check box->Edit field->List items..etc.

**Expected behavior**
After invoking the 'Columns' control, Keyboard tab focus order should be in sequential while navigating using TAB key. After invoking 'Columns' control, Keyboard tab focus should be as below

Invoke columns control->columns popup displays->Focus should be on Columns control/first interactive(i.e, chevron) in columns popup.

**Please tell us about your environment:**
**OS Version:** 22H2 OS Build 22621.1265
**AG Grid version:** 25.1.0
**Browser:** Edge(Chromium) Version 110.0.1587.63 (Official build) (64-bit)
**Language:** ES6

https://user-images.githubusercontent.com/93735775/223486690-efb43c4d-8850-461b-a958-60ddbd850f16.mp4

**Comment by kiril-matev** [comment]
Hello @msftedad,
 
Thank you for using AG Grid and for bringing this up.
 
Please note that the columns tool panel is an AG Grid Enterprise feature and can only be used with an AG Grid Enterprise license. Once you have a license you have access to our support system, where we can respond more quickly and provide support. Please contact our sales team to get access to our support portal.
 
We've addressed this keyboard navigation issue in AG Grid v29.0 and later. 

I've upgraded your sample to v29.0 and you can run the same test case to confirm that keyboard focus is handled correctly in the columns tool panel:
1. Open
https://codesandbox.io/s/react-functional-component-forked-0sx8kn?file=/package.json
2. Click the right-most cell in the bottom grid row
3. Press TAB
Actual & Expected: Columns button focused in sidebar
4. Press ENTER key
Actual & Expected: Columns tool panel shown, Columns button focused in sidebar
5. Press TAB key
Actual & Expected: Pivot mode toggle focused
6. Press TAB key
Actual & Expected: Search item focused
7. Press TAB key
Actual & Expected: Search checkbox focused
8. Press TAB key
Actual & Expected: Search input focused
9. Press TAB key
Actual & Expected: Top column list item focused (ATHLETE)
10. Press TAB key
Actual & Expected: Column list item focused (AGE)
11. Press TAB key
Actual & Expected: Column list item focused (COUNTRY)
12. Press TAB key
Actual & Expected: Focus leaves the grid, input below grid focused
 
Please upgrade to AG Grid 29.0 or later to resolve this issue.
 
Thank you for your time!

**Comment by msftedad** [comment]
Hi Thank you for the response @kiril-matev, we will check if we can reproduce the issue in AG Grid 29.0 or later and get back to you.
